openerp/addons/base/ir/ir_ui_view.py

```diff
@@ -662,7 +662,7 @@ def check_group(node):
             if node.get('groups'):
                 can_see = self.user_has_groups(
                     cr, user, groups=node.get('groups'), context=context)
-                if not can_see:
+                if not can_see or (request and not request.debug and 'base.group_no_one' in node.get('groups')):
```

Comment:
Doesn't seem right; technical != bypassing groups validation. What are you trying to do here precisely ?